openembedded-core-mbl/scripts/signctl/README.md

```diff
@@ -3,31 +3,32 @@
 ## Overview
 
 This signing tool is an early release and is still under development. It has the following restrictions:
-* The tool has only been tested with the Raspberry Pi 3B+
-* The tool uses Hashicorp Vault as an example back-end for the private key storage and signing operations.
-* The tool only supports signing for a single device at the moment.
+* The tool has been tested with the Raspberry Pi 3B+, PICO PI with IMX7D SoM, PICO-PI with IMX6UL SoM and NXP 8M Mini EVK devices.
+* The tool uses Hashicorp Vault as an example backend for the private key storage and signing operations.
+* The tool only supports signing for a single device at the moment. Although a multi-device flow is possible if you configure a separate Vault instance for each device.
 
-The release version of the tool will support multiple devices; we will provide documentation and guidance for setting access control policies in Vault, and for locking down the Vault machine for production use. **The current flow is only for development use.**
+The release version of the tool will provide full support for multiple devices; we will also provide documentation and guidance for setting access control policies in Vault and for locking down the Vault machine for production use. **The current flow is only for development use.**
 
-The `signctl` tool automates the basic signing flow for Raspberry PI 3. For more information on the background and details of the signing operations see [the basic signing flow document][mbl-basic-signing-flow].
+The `signctl` tool automates the signing flow for TF-A up to BL2. For more information on the background and details of the signing operations see [the basic signing flow document][mbl-basic-signing-flow].
```

Comment:
TF-A up to BL2?  It includes the FIP aswell which is bootloader component 2.

Reply:
It was meant to mean TF-A including FIP and BL2. When I refer to BL2 in this document I mean TF-A BL2 and not our "bootloader component 2" which is the FIP. I explain this near the link to the update document. I've re-worded this in 8af62db in an attempt to clarify.